.env

```diff
@@ -1,3 +1,3 @@
-KNUU_TIMEOUT=60m # The timeout for the tests
+KNUU_TIMEOUT=120m   # The timeout for the tests
```

Comment:
The old version works with this, but the new version of knuu receives it as a parameter so user can pass the duration they need.

Reply:
how does the user pass this param?